pylint/test/functional/inconsistent_returns.py

```diff
@@ -158,6 +165,11 @@ def not_consistent_returns_inner(arg): # [inconsistent-return-statements]
                 return arg
     return not_consistent_returns_inner
 
+def bug_1771_counter_example(var): # [inconsistent-return-statements]
```

Comment:
Can you also add a test where the never returning function is customized through a rcfile? Check in `functional` directory, we have a couple of those where the option is defined in a `.rc`file